src/components/TripCard/TripCard.spec.js

```diff
@@ -1,13 +1,30 @@
 import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
 import TripCard from './TripCard'
 
 describe('TripCard', () => {
-  it('renders the heading, the start and the end date', () => {
+  it('renders the heading, the start and the end date and the delete button', () => {
     render(
       <TripCard city="New York" startDate="2021-03-15" endDate="2021-03-24" />
     )
     expect(screen.getByText('New York')).toBeInTheDocument()
     expect(screen.getByText('2021 03/15')).toBeInTheDocument()
     expect(screen.getByText('2021 03/24')).toBeInTheDocument()
+    expect(screen.getByText(/Delete/)).toBeInTheDocument()
+  })
+
+  it('calls onClick when clicked', () => {
+    const callback = jest.fn()
+    render(
+      <TripCard
+        city="New York"
+        startDate="2021-03-15"
+        endDate="2021-03-24"
+        onClick={callback}
```

Comment:
You don't have a `onClick` as props in your component. So this line won't do anything. You want to test whether the function `onDelete` you call on a click of the button will be triggered or not.
```suggestion

```

Reply:
Your post in the slack chat was very helpful! Thank you 🚀